intrastat_product_hscodes_import/wizards/intrastat_hscodes_import_installer.py

```diff
@@ -80,9 +80,7 @@ def execute(self):
                 continue
             lang_found = True
             CN_fn = [x for x in CN_fns if x[5:7] == lang][0]
-            with io.open(
```

Comment:
changing the encoding from windows-1252 to UTF-8 will break the fr and de translations